### PR TITLE
Bug 1188591 - SmartOn: Cybersecurity

### PR DESCRIPTION
**Pull request by craigcook**
This depends on #3443 hence the extra commit. That PR should merge first, then this will need rebasing.

### CODE REVIEW

File: bedrock/teach/templates/teach/smarton/base.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import share_cta with context %}
+
+{% extends "base-resp.html" %}
```
Comment (flodolo) [inline bedrock/teach/templates/teach/smarton/base.html]: We should load `teach/smarton/index` as lang file

File: bedrock/teach/templates/teach/smarton/security.html [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+        <p>
+          {{ _('If a link or download doesn’t feel quite right, it probably isn’t.') }}
+          {{ _('Make sure you install programs only from verified trusted sources, and don’t click on anything that looks even remotely suspicious – even when you think you know the source.') }}
+        </p>
+      </div>
+
+      <div class="tip">
+        <h4>{{ _('Watch yourself in public') }}</h4>
+        <p>
+          {{ _('When you navigate the Web on a public WiFi network, anyone who chooses to snoop can see some or all of your activity.') }}
```
Comment (flodolo) [inline bedrock/teach/templates/teach/smarton/security.html]: Isn't the proper spelling Wi-Fi?

File: bedrock/teach/templates/teach/smarton/security.html [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+        {{ _('The more we talk, share tips and stay alert about cybersecurity, the safer we all are.') }}
+        {{ _('Keep the conversation going with your community on social media, at home — maybe even on that public Wi-Fi at the airport.') }}
+      </p>
+
+      <aside class="callout">
+        <p>{{ _('Connect with your community to explore topics related to privacy and tracking by joining or starting a <a href="{url}">Mozilla Club</a>.').format(url='https://teach.mozilla.org/clubs/')|safe }}</p>
+      </aside>
+
+      <aside class="callout">
+        <p>{{ _('Are you a teacher? Give your students a <a href="{url}">simple lesson on government surveillance</a>.').format(url='https://teach.mozilla.org/activities/')|safe }}</p>
```
Comment (flodolo) [inline bedrock/teach/templates/teach/smarton/security.html]: Is this correct? Government Surveillance in the CyberSecurity page?

Comment (flodolo) [inline bedrock/teach/templates/teach/smarton/security.html]: Checked the copy deck, it should be "Are you a teacher? Give your students a simple lesson in cybersecurity."

File: bedrock/teach/templates/teach/smarton/security.html [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+            <p>
+              {{ _('What kind of security tips and tricks do you use online?') }}
+              <a class="tweet-this" href="{{ 'https://www.twitter.com/intent/tweet?url=%s&amp;text=%s'|format('https://mzl.la/foo'|urlencode, _('What kind of security tips and tricks do you use online?')|urlencode)|e }}">{{ _('Tweet this') }}</a>
+            </p>
+            {% include 'teach/smarton/includes/list-number-2.svg' %}
+          </li>
+          <li>
+            <p>
+              {{ _('On a scale from 1 to 10, how safe do you feel online?') }}
+              {{ _('Is it always clear to you whether or not your activities are secure?') }}
```
Comment (flodolo) [inline bedrock/teach/templates/teach/smarton/security.html]: Sorry, missed this one: it's a tweet, let's keep it in one string for sake of clarity and shortness.
```
{{ _('On a scale from 1 to 10, how safe do you feel online? Is it always clear to you whether or not your activities are secure?') }}
```

File: bedrock/teach/templates/teach/smarton/security.html [hunk 5]
```diff
@@ -0,0 +1,324 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "teach/smarton/base.html" %}
+
+{% block page_title %}{{ _('Get smart on cybersecurity') }}{% endblock %}
+{% block page_desc %}{{ _('Look both ways, stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity.') }}{% endblock %}
```
Comment (craigcook) [inline bedrock/teach/templates/teach/smarton/security.html]: String added 20 October: "Look both ways, stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity."

**Comment by craigcook** [comment]
Closing this PR in favor of #3524 from the Mozilla repo.

**Pull request closed without merging**